test/adapters/kagome/cmake/HunterPasswords.cmake

```diff
@@ -0,0 +1,11 @@
+hunter_upload_password(
+  # REPO_OWNER + REPO = https://github.com/forexample/hunter-cache
+  REPO_OWNER "soramitsu"
+  REPO "hunter-binary-cache"
+
+  # USERNAME = warchant
+  USERNAME "$ENV{GITHUB_HUNTER_USERNAME}"
+
+  # PASSWORD = GitHub token saved as a secure environment variable
+  PASSWORD "$ENV{GITHUB_HUNTER_TOKEN}"
+)
```

Comment:
Same here. And we have no control over this repo. I am sure hunter can be configured in a similar fashion just [with  environmental variables](https://hunter.readthedocs.io/en/latest/reference/user-variables.html) to set this directly with a few exports and no upstream changes.